contracts/V1/DataTypes.sol

```diff
@@ -31,13 +31,13 @@ struct ForkDataAvailabilityKey {
 }
 
 struct SignedBlock {
-    bytes encodedBlock; //TODO! change this to bytes
-    bytes signature; //TODO! change this to bytes
+    bytes encodedBlock;
+    bytes[] signatures;
 }
 
 struct ConfirmedBlock {
     bytes encodedBlock;
```

Comment:
Let's change encodedBlock to signedBlock, so it's not possible to present a blockConfirmation without the original signature